ports/ngspice/portfile.cmake

```
@@ -25,14 +25,15 @@ vcpkg_add_to_path(PREPEND "${BISON_DIR}")
 # Sadly, vcpkg globs .libs inside install_msbuild and whines that the 47 year old SPICE format isn't a MSVC lib ;)
 # We need to kill them off first before the source tree is copied to a tmp location by install_msbuild
 
-file(REMOVE_RECURSE ${SOURCE_PATH}/contrib)
-file(REMOVE_RECURSE ${SOURCE_PATH}/examples)
-file(REMOVE_RECURSE ${SOURCE_PATH}/man)
-file(REMOVE_RECURSE ${SOURCE_PATH}/tests)
+file(REMOVE_RECURSE "${SOURCE_PATH}/contrib")
+file(REMOVE_RECURSE "${SOURCE_PATH}/contrib")
+file(REMOVE_RECURSE "${SOURCE_PATH}/examples")
+file(REMOVE_RECURSE "${SOURCE_PATH}/man")
+file(REMOVE_RECURSE "${SOURCE_PATH}/tests")
```

```suggestion
file(REMOVE_RECURSE "${SOURCE_PATH}/contrib")
file(REMOVE_RECURSE "${SOURCE_PATH}/examples")
file(REMOVE_RECURSE "${SOURCE_PATH}/man")
file(REMOVE_RECURSE "${SOURCE_PATH}/tests")
```
 Remove duplicated line

---

Thanks for pointing it out.